internal/config/backup_test.go

```diff
@@ -48,44 +49,79 @@ func TestBackupManager_Bind(t *testing.T) {
 		return nil
 	}
 	tests := []test{
-		// TODO test cases
-		/*
-		   {
-		       name: "test_case_1",
-		       fields: fields {
-		           Host: "",
-		           Port: 0,
-		           Client: GRPCClient{},
-		       },
-		       want: want{},
-		       checkFunc: defaultCheckFunc,
-		   },
-		*/
-
-		// TODO test cases
-		/*
-		   func() test {
-		       return test {
-		           name: "test_case_2",
-		           fields: fields {
-		           Host: "",
-		           Port: 0,
-		           Client: GRPCClient{},
-		           },
-		           want: want{},
-		           checkFunc: defaultCheckFunc,
-		       }
-		   }(),
-		*/
+		{
+			name: "return Backup when client is nil",
+			fields: fields{
+				Client: nil,
+			},
+			want: want{
+				want: &BackupManager{
+					Client: newGRPCClientConfig(),
+				},
+			},
+		},
+		func() test {
+			c := &GRPCClient{
+				HealthCheckDuration: "1s",
+			}
+			return test{
+				name: "return Backup when client is not nil",
+				fields: fields{
+					Client: c,
+				},
+				want: want{
+					want: &BackupManager{
+						Client: c.Bind(),
+					},
+				},
+			}
+		}(),
+		func() test {
+			k := "ADDR"
+			v := "http://backupmanager.com"
+			c := &GRPCClient{
+				Addrs: []string{v},
+			}
+			return test{
+				name: "return Backup when addrs is set via environment variable",
+				fields: fields{
+					Client: &GRPCClient{
+						Addrs: []string{
+							"_" + k + "_",
+						},
+					},
+				},
+				beforeFunc: func(t *testing.T) {
+					t.Helper()
+					if err := os.Setenv(k, v); err != nil {
+						t.Fatal(err)
+					}
+				},
+				afterFunc: func(t *testing.T) {
+					t.Helper()
+					if err := os.Unsetenv(k); err != nil {
+						t.Fatal(err)
+					}
+				},
+				want: want{
+					want: &BackupManager{
+						Client: c.Bind(),
+					},
+				},
```

Comment:
I think you should not use `c.Bind()` for the assertion.

Reply:
ah, sorry it is my mistake. please ignore

Reply:
fixed, thank you.

Reply:
sorry I start thinking we should ignore this suggestion, because it adds unnecessary complexity to the test implementation. 
@vankichi @hlts2 What do you think?